### ISSUE TITLE
[Live Trains] Blackbox Landscape LCD Formatting Issue

### ISSUE THREAD

**Issue by opott** [davwheat/rail-announcements#162]
To my knowledge, the calling points are supposed to display in pages (usually 3 at a time, unless they don't fit) rather than being scrolling text.

**Comment by davwheat** [comment]
Yes, this changed between my first visit to record footage of the boards at Perry Barr and present-day.
